app/src/main/java/org/oppia/android/app/settings/profile/ProfileRenameActivity.kt

```diff
@@ -15,6 +16,8 @@ class ProfileRenameActivity : InjectableAppCompatActivity() {
   lateinit var profileRenameActivityPresenter: ProfileRenameActivityPresenter
 
   companion object {
+
+    /** Creates the Intent for [ProfileRenameActivity]. */
```

Comment:
```suggestion
    /** Returns an [Intent] for opening [ProfileRenameActivity]. */
```

Ditto for other 'createIntent'-esque methods.

Reply:
done